release-notes/3.0/preview/api-diff/preview4/standalone-packages/3.0-preview4-standalone-packages_System.Data.md

```diff
@@ -0,0 +1,124 @@
+# System.Data
+
+``` diff
+ namespace System.Data {
+-    public static class DataRowComparer {
```

Comment:
We removed APIs here? This doesn't seem correct.

Reply:
Or this highlighting that these APIs are now "in-box"?

Reply:
Yes to the latter. IIRC, we first moved the implementation of all thes types from System.Data.DataSetExtensions into System.Data.Common which is in-box, and then we also moved the System.Data.DataSetExentions facade to in-box as well.